### ISSUE TITLE
Staghorn calculus

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6649]
**Preferred term label:**
Staghorn calculus

**Synonyms**
Staghorn renal stone

**Definition (free text, please give PubMed ID)**
Large branching stones that fill part of all of the renal pelvis and renal calyces and they can be complete or partial depending on the level of occupancy of the collecting system.
PMID: 29658394

**Parent term (use hpo.jax.org/app)**
Nephrolithiasis

**Comment by pnrobinson** [comment]
added term
